.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: node_js
+script: echo "It's alright!"
```

Comment:
I can't answer your first question, but about your second

> And does it mean that I should not use rebase to get a feature-branch "updated"

No, in fact rebase is perfect to get a feature-branch updated.